pkg/parent/warn.go

```diff
@@ -0,0 +1,64 @@
+package parent
+
+import (
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/rootless-containers/rootlesskit/pkg/parent/mount"
+)
+
+func warnPropagation(propagation string) {
+	mounts, err := mount.GetMounts()
+	if err != nil {
+		logrus.Warn("Failed to parse mountinfo")
```

Comment:
are we interested in the error as part of warnings? If so, perhaps `logrus.WithError(err).Warn(...`

Reply:
done